File: src/main/java/com/example/desafio/api/models/repositorys/ModelCarRepository.java
package com.example.desafio.api.models.repositorys;

import com.example.desafio.api.models.entitys.ModelCarEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ModelCarRepository extends JpaRepository<ModelCarEntity, Long> {
    ModelCarEntity findByModelName(String modelName);
}

File: src/main/kotlin/com/example/desafio/api/models/repositorys/ModelCarRepository.kt
package com.example.desafio.api.models.repositorys

import com.example.desafio.api.models.entitys.ModelCarEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ModelCarRepository : JpaRepository<ModelCarEntity?, Long?> {
    fun findByModelName(modelName: String?): ModelCarEntity?
}